scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -525,59 +525,68 @@ Called SNVs:
 Variant effect prediction (*Ensemble VEP output*):  
 [Download VEP_out.csv](`r sprintf('data:text/csv;base64,%s', vep)`)
 
-```{r csv_output_variant_plot, include = F, eval = executeDeconvolution}
```

Comment:
is there any use for executeDeconvolution left?  If not: has it been deleted elsewhere?

Reply:
this and the following issues are captured in https://github.com/BIMSBbioinfo/pigx_sars-cov-2/issues/114

Reply:
will resolve this for now, since @jonasfreimuth is going to work on this file more extensively